src/connectedmachine/setup.py

```diff
@@ -10,7 +10,7 @@
 from setuptools import setup, find_packages
 
 # HISTORY.rst entry.
-VERSION = '0.7.0'
+VERSION = '0.8.0'
```

Comment:
please correct version to be 1.0.0b1 as extension versioning schema bot suggested 
For more info about extension versioning, please refer to [Extension version schema](https://github.com/Azure/azure-cli/blob/release/doc/extensions/versioning_guidelines.md)

Reply:
fixed